test/snippet-tests.js

```diff
@@ -24,4 +24,12 @@ describe('Snippet', function() {
     assert.deepEqual(identify._q[2], ['add', 'key3', 2]);
     assert.deepEqual(identify._q[3], ['setOnce', 'key4', 'value2']);
   });
+
+  it('amplitude object should proxy Revenue object and calls', function() {
+    var revenue = new amplitude.Revenue().setProductId('productIdentifier').setQuantity(5).setPrice(10.99);
```

Comment:
Here we test that the snippet correctly creates the proxy revenue object, and it captures the function calls into _q queue